site/profile/manifests/core/common.pp

```diff
@@ -8,12 +17,15 @@
   include sysstat
   include epel
   include sudo
-  include accounts
```

Comment:
removing the `accounts` mod here without re-adding it elsewhere will cause a massive amount of breakage.

Reply:
Good catch, this came from an aborted refactor.